winui/Kanban/Getting-started.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: Getting Started with WinUI Kanban control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Kanban (SfKanban) control, its elements, and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+# Getting Started with WinUI Kanban (SfKanban)
```

Comment:
Getting Started with WinUI Kanban Control

Reply:
changed to Getting Started with WinUI Kanban Control